lib/allocation.c

```diff
@@ -1446,7 +1446,7 @@ hw_alloc_reset(const enum pqos_cdp_config l3_cdp_cfg,
 		for (j = 0; j < l3cat_id_num; j++) {
                         unsigned core = 0;
 
-			ret = pqos_cpu_get_one_core(m_cpu, l3cat_ids[j], &core);
+			ret = pqos_cpu_get_one_by_l3cat_id(m_cpu, l3cat_ids[j], &core);
```

Comment:
Travis reported the following code style issue:

allocation.c:1449: WARNING: line over 80 characters
#1449: FILE: allocation.c:1449:
+			ret = pqos_cpu_get_one_by_l3cat_id(m_cpu, l3cat_ids[j], &core);